miri-script/src/commands.rs

```diff
@@ -19,6 +20,7 @@ fn detect_miri_dir() -> std::io::Result<PathBuf> {
     Ok(PathBuf::from(MIRI_SCRIPT_ROOT_DIR).canonicalize()?.parent().unwrap().into())
 }
 
+/// Needed for `./miri bench`.
 fn get_current_toolchain(miri_dir: &Path) -> std::io::Result<String> {
```

Comment:
get_current_toolchain isn't just used in `./miri bench`, something went wrong with comment porting here

Reply:
Yep; the original comment was for a TOOLCHAIN variable. I guess we could document `active_toolchain` member variable or scrap this altogether?

Reply:
The original comment was for evaluation order: `TOOLCHAIN` needs to be set before the early block of commands, because of `bench`. The comment doesn't document the variable, it documes where the variable is computed.

With the Rust version, each command just queries the toolchain as it sees fit (right?), so this entire order issue disappears.

Having a doc comment that describes what this fn does would be useful though.